.gitignore

```diff
@@ -18,5 +18,7 @@ debug
 .vs
 
 .idea/
+/*.iml
 
-_tiltbuild
\ No newline at end of file
+_tiltbuild
+/coverage.out
```

Comment:
This file is not related to this PR description change.

Reply:
Change removed from this PR